### PR TITLE
Add firefox/browsers/chromebook (#9490 Fixes #9512, #9515)

### LINKED ISSUE #9512: Epic: Chromebook page
## Description

We want to build a page for chromebook

url: /firefox/browsers/chromebook/

Pitch: https://docs.google.com/document/d/17aL69fIIUhftGhFnx87OcfYcWoecs-xno1a1rT6_Nc4
Gdrive: https://drive.google.com/drive/folders/1VFoA7IJlj_jUYWkGIp4kYhMGHRjzJ-k_

---

#### Issues
- [x] Content https://github.com/mozilla/bedrock/issues/9513
- [x] Design  https://github.com/mozilla/bedrock/issues/9514
- [ ] Coding https://github.com/mozilla/bedrock/issues/9515

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Adds firefox/browsers/chromebook

## Issue / Bugzilla link
#9490
#9512
#9515

## Testing
[Copy](https://docs.google.com/document/d/19Vin0P4F8w2FdeXGl8Df6wSLIzD2bKPAcD5XRcLTc-0/)
[Design](https://github.com/mozilla/bedrock/issues/9514#issuecomment-706293652) -- _design reflects older menu list design_

### CODE REVIEW

**Comment by amychurchwell** [comment]
Marking as "Do Not Merge" until demo received approvals from Raphael & Chad

**Comment by amychurchwell** [comment]
Approved!

File: bedrock/firefox/templates/firefox/browsers/chromebook.html
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/browsers/chromebook.html:39]: Can you double-check the analytics on this link, please?

File: bedrock/firefox/templates/firefox/browsers/chromebook.html
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/browsers/chromebook.html:79]: Do we want analytics on this link? I'd do it like this: https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/browsers/compare/brave.html#L86

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
Looking good; just needs a couple finishing touches.

The card images are larger than they need to be. Can you please create high and low res versions? The high res shouldn't be wider than 1200px. ([image sizes](https://protocol.mozilla.org/patterns/molecules/card.html) are defined in Protocol)

File: l10n/en/firefox/browsers/chromebook.ftl
Comment (alexgibson) [inline l10n/en/firefox/browsers/chromebook.ftl:9]: Fluent IDs should use a page specific naming prefix: https://bedrock.readthedocs.io/en/latest/l10n.html#ftl-string-ids

File: l10n/en/brands.ftl
Comment (alexgibson) [inline l10n/en/brands.ftl:132]: Should it be `Chrome OS`? https://www.google.com/intl/en_uk/chromebook/chrome-os/

File: l10n/en/firefox/browsers/chromebook.ftl
Comment (alexgibson) [inline l10n/en/firefox/browsers/chromebook.ftl:28]: Remove /en-US/

File: bedrock/firefox/templates/firefox/browsers/chromebook.html
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/browsers/chromebook.html:79]: I'm not sure, I just included it to be safe. Happy to remove if no one will use the info?

Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/browsers/chromebook.html:79]: asking Raphael.

File: l10n/en/firefox/browsers/chromebook.ftl
Comment (amychurchwell) [reply l10n/en/firefox/browsers/chromebook.ftl:9]: adding `browsers-chromebook-` at the beginning of all IDs

File: bedrock/firefox/templates/firefox/browsers/chromebook.html
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/browsers/chromebook.html:38]: Not sure about the lone `data-display-name` property here without any others. I think we normally use this on download links?

File: bedrock/firefox/templates/firefox/browsers/chromebook.html
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/browsers/chromebook.html:75]: You need to define android_url (I keep making this mistake myself 😆 )

`{% set android_url = firefox_adjust_url('android', 'firefox-daylight-promo-banner') %}`

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
Updates are looking 💯 but I found something we should have caught in an earlier review.

**Comment by amychurchwell** [comment]
@stephaniehobson good catch! thank you. 👍

**Review by stephaniehobson — APPROVED** [review]
R+ 📘

**Pull request merged**
